charts/dsp-atlantis/templates/azureCredentials.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.azure.enabled -}}
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: {{ .Values.name }}-azure-creds-secretdefinition
+  labels:
+    {{- template "dsp-atlantis.labels" }}
+spec:
+  name: azure-cert
+  keysMap:
+    {{ .Values.azure.certificateFile }}:
+      path: {{ required "A valid .Values.vault.azure.pathPrefix is required" .Values.vault.azure.pathPrefix }}/certificate
+      encoding: base64
+      key: {{ .Values.azure.certificateFile }}.b64
+---
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: {{ .Values.name }}-azure-env-secretdefinition
+  labels:
+    {{- template "dsp-atlantis.labels" }}
+spec:
+  name: azure-env
+  keysMap:
+    ARM_CLIENT_CERTIFICATE_PASSWORD:
+      path: {{ required "A valid .Values.vault.azure.pathPrefix is required" .Values.vault.azure.pathPrefix }}/atlantis/env
+      key: ARM_CLIENT_CERTIFICATE_PASSWORD
+    ARM_CLIENT_ID:
+      path: {{ required "A valid .Values.vault.azure.pathPrefix is required" .Values.vault.azure.pathPrefix }}/atlantis/env
+      key: ARM_CLIENT_ID
+    ARM_TENANT_ID:
+      path: {{ required "A valid .Values.vault.azure.pathPrefix is required" .Values.vault.azure.pathPrefix }}/atlantis/env
+      key: ARM_TENANT_ID
```

Comment:
Does this replace [https://github.com/broadinstitute/terra-helm-definitions/blob/master/dsp-tools/dsp-atlantis/values.yaml#L46](https://github.com/broadinstitute/terra-helm-definitions/blob/master/dsp-tools/dsp-atlantis/values.yaml#L46) or is it creating the secrets that file references?

Reply:
It creates the secrets referenced in that file

Reply:
rather than me creating them manually